qtconsole/rich_jupyter_widget.py

```diff
@@ -140,9 +140,8 @@ def _handle_execute_result(self, msg):
                 self._pre_image_append(msg, prompt_number)
                 try:
                     self._append_latex(data['text/latex'], True)
-                except Exception:
-                    self.log.error("Failed to render latex: '%s'", data['text/latex'], exc_info=True)
```

Comment:
> Isn't it a good thing to log failure to render the latex?

Not for Spyder and Canopy because those failures are shown in the terminal where those applications were started. If they started graphically, the errors won't be seen by their users.

Besides, (as I said in ipython/ipython#9203) possible errors are:

1. Matplotlib is unable to handle Latex, something not easily discernible from the traceback.
2. Latex has syntax errors, something that also generates one of those long and almost unreadable report errors.

So this simplifies things and handles all cases I mentioned at the beginning correctly.